.github/workflows/docker.yml

```
@@ -0,0 +1,23 @@
+name: Docker builds
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

@mlxd does it make sense to run this on all PRs? Since the tests are already being run on PRs.

Maybe it makes sense to only run this on the `master` branch?
```suggestion
  push:
    branches:
      - master
```